heroku_connect/contrib/health_check.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from health_check.backends import BaseHealthCheckBackend
+from health_check.exceptions import ServiceUnavailable
+
+from ..conf import settings
+from ..utils import get_connection_id, get_connection_status
+
+logger = logging.getLogger('heroku_connect')
```

Comment:
Sorry I already changed it, but it might makes sense to get the `health-check` logger here